clusterloader2/testing/batch/job.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.Name}}
+  labels:
+    group: test-job
+spec:
+  parallelism: {{.Replicas}}
+  completions: {{.Replicas}}
+  template:
+    metadata:
+      labels:
+        group: test-pod
+    spec:
+      containers:
+      - name: {{.Name}}
+        image: bash
```

Comment:
Yes, I plan to change this to a proper load test. But I still would like to have this merged as it serves as proof that the measurement works.

I will request to upload an image soon. I was told I need to verify the license. I might create a simple go code that just exits, otherwise.

I don't think we need sleep for now, although that could be an argument for the command.